app/Http/Controllers/PostController.php

```diff
@@ -71,6 +71,8 @@ public function edit($id)
   public function update(CreateUpdatePostRequest $request)
   {
 
+    
+    
```

Comment:
remove white spaces, you must to take on mind of important is the indentation when you coding